ember-flight-icons/addon/components/flight-icon.hbs

```diff
@@ -1,6 +1,6 @@
 <svg 
   aria-hidden="true"
-  class="flight-icon icon-{{@name}} {{if this.display "display-inline"}}"
+  class="flight-icon icon-{{@name}} {{if this.display "display-inline"}} {{if this.isDemo "demo-icon"}} {{if this.isDemoHidden "d-none"}}"
```

Comment:
I would concerned about adding public API/logic like this to the component that is only meant to support the demo app. 

It also seems like this might be an example of something that is going to be common with consumers (wanting to add a class to the svg) which would be good for us to come up with a canonical solution for. What do you think?

Reply:
@Dhaulagiri I think you are correct. We could position the class attribute such that using `...attributes` would disallow addition of any other attribute but appended class names.

Reply:
Updated; issue resolved.